ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -64,6 +67,18 @@ public void setText(ReactTextUpdate update) {
       (int) Math.floor(update.getPaddingRight()),
       (int) Math.floor(update.getPaddingBottom()));
 
+    float nextLetterSpacing = update.getLetterSpacing();
+    int fontSize = update.getFontSize();
+    if (!FloatUtil.floatsEqual(mLetterSpacing, nextLetterSpacing)) {
+      mLetterSpacing = nextLetterSpacing;
+      if(Float.isNaN(mLetterSpacing)) {
+        setLetterSpacing((float)0.0);
+      } else {
+        //calculate EM from proper font pixels
+        setLetterSpacing(1+(mLetterSpacing-PixelUtil.toDIPFromPixel(fontSize))/PixelUtil.toDIPFromPixel(fontSize));
```

Comment:
I think `setLetterSpacing` isn't available on all android versions. Can you run this in Android 4.x and see if everything builds? We don't need the letter spacing to work in that case, but it shouldn't break anything either.